test/integration/src/py/test_rollback_chain.py

```diff
@@ -30,7 +30,8 @@ def test_rollback_chain(source_ethereum_address):
         ethereum_private_key_env_var="ETHEREUM_PRIVATE_KEY",
         bridgebank_address=get_required_env_var("BRIDGE_BANK_ADDRESS"),
         ethereum_network=(os.environ.get("ETHEREUM_NETWORK") or ""),
-        amount=amount
+        amount=amount,
```

Comment:
Maybe this is a failure of imagination on my part, but I don't know how we could test rolling things back on a testnet we don't have control over.

Reply:
There's no code yet to automate running tests against another chain.  This PR is just to make it possible for a human to follow the instructions in the readme to run tests that they think are sensible against something other than a local ganache test setup.